test/e2e/test_app.py

```diff
@@ -94,17 +93,17 @@ def test_complete_app(browser, workflow_with_customizations, testing_env_variabl
 
     # Configure translate language to en-ES
     
-    translate_languages_box = browser.find_element_by_xpath("/html/body/div/div/div[2]/div[2]/div/div[1]/div[3]/div[2]/fieldset/div/div[3]/fieldset/div/div[2]/div[1]/p/input")
+    translate_languages_box = browser.find_element_by_xpath("/html/body/div/div/div[2]/div[2]/div/div[1]/div[3]/div[2]/fieldset/div/div[2]/fieldset/div/div[2]/div[1]/p/input")
```

Comment:
This is basically a unit test for the [VoerroTagsInput](https://github.com/voerro/vue-tagsinput) component. Did you intend to test functionality of an external package? 

It's good practice to write unit tests for dynamically generated output, and voerrotagsinput dynamically generates output, but we did not write voerrotagsinput. Any failures in this test will likely be failures in the test, not actual bugs in voerrotagsinput, as demonstrated by the xpath change  here.